packages/gatsby-source-shopify/src/nodes.js

```diff
@@ -129,6 +138,9 @@ export const ProductNode = imageArgs =>
     return node
   })
 
+export const ProductMetafieldNode = _imageArgs =>
```

Comment:
Not really new here -- but I wonder why this pattern is used through all of these node types?

Oh well!

Reply:
That's a really good question - I didn't want to reinvent anything, so I just followed the same pattern as before. I don't totally understand it to be entirely honest.

Reply:
🤷‍♂ Your guess is as good as mine. Not hurting anything -- we'll clean it up later!